pbsmetrics/go_metrics.go

```diff
@@ -120,11 +123,15 @@ func NewBlankMetrics(registry metrics.Registry, exchanges []openrtb_ext.BidderNa
 		ImpsTypeAudio:  blankMeter,
 		ImpsTypeNative: blankMeter,
 
-		AdapterMetrics: make(map[openrtb_ext.BidderName]*AdapterMetrics, len(exchanges)),
-		accountMetrics: make(map[string]*accountMetrics),
+		AdapterMetrics:  make(map[openrtb_ext.BidderName]*AdapterMetrics, len(exchanges)),
+		accountMetrics:  make(map[string]*accountMetrics),
+		MetricsDisabled: config.DisabledMetrics{},
```

Comment:
If the DisabledMetrics was not a pointer, we would not need any nil checking logic below.

Reply:
It does make sense. I can get rid of it if you want.

Reply:
I think it will be cleaner to not have a pointer. Copying a handfull of booleans on startup should not be a big performance issue.